apps/frontend/src/pages/quiz/libs/components/analyzing/analyzing.tsx

```diff
@@ -1,11 +1,8 @@
 import { Button } from "~/libs/components/components.js";
 
+import { type Properties } from "../../types/types.js";
```

Comment:
Properties should be in component. This will help us if we want to change properties for some screens in future